app/scripts/components/common/layout-root/index.tsx

```diff
@@ -35,17 +43,58 @@ const PageBody = styled.div`
 `;
 
 function LayoutRoot(props: { children?: ReactNode }) {
+  const useConsentForm = checkEnvFlag(process.env.COOKIE_CONSENT_FORM);
```

Comment:
this is still using env variable. And I think your intention is to depend on veda config instead of env var?